content/en/docs/concepts/cluster-administration/flow-control.md

```diff
@@ -782,6 +782,100 @@ APF adds the following two headers to each HTTP response message.
 - `X-Kubernetes-PF-PriorityLevel-UID` holds the UID of the
   PriorityLevelConfiguration object associated with that FlowSchema.
 
+## APF Best Practices
```

Comment:
```suggestion
## Good practice for using API priority & fairness
```
We recommend _good_ practice, not _best_ practice. It lets us be humble about whether there is an even better way out there.